demo-angular/app/App_Resources/Android/app.gradle

```diff
@@ -0,0 +1,16 @@
+// Add your native dependencies here:
+
+// Uncomment to add recyclerview-v7 dependency
+//dependencies {
+//	compile 'com.android.support:recyclerview-v7:+'
+//}
+
+android {  
+  defaultConfig {  
+    generatedDensities = []
+    applicationId = "org.nativescript.demo"  
```

Comment:
@jannotabamo could you change the id to `org.nativescript.demotwilioangular`? I would need a specific Id for setup the Push Notifications on iOS

Reply:
@msaelices done 👍